package-lock.json

```diff
@@ -1351,7 +1351,7 @@
     },
     "fs-extra": {
       "version": "2.0.0",
-      "resolved": "https://registry.npmjs.org/fs-extra/-/fs-extra-2.0.0.tgz",
+      "resolved": "https://nexus3.linkurious.net/repository/npm/fs-extra/-/fs-extra-2.0.0.tgz",
```

Comment:
@yigitakbulut can you revert that? this is a public github repo, it should not reference our private npm repo